packages/cli/bin/run

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+
+require('@oclif/command')
```

Comment:
this is the new entrypoint for the CLI. All of the code currently in `entry.js` (startup checks, etc) could either go in an [init hook](https://oclif.io/docs/hooks) or get called in a function from `BaseCommand.constructor`

Reply:
Is `bin/run` the one we'll have to specify as `main` in `package.json`?

Reply:
yep!